open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+You start simple.
 
 ### Thinking through outcomes
 
-What outcomes will our flowchart have?
+What outcomes will our flowchart have? 
 
 1. Go to the party
 2. Stay home
-3. Go, but don't stay too late
-
-### Gathering the facts
-
-What kind of information do we want to know to advise our friends? Let's think of some possible questions.
-
-* Do you like parties?
-* Is there anyone there who you want to meet?
-* How difficult is it to get to the party?
-* Do you have any big assignments or studying you should do?
-
-### Sorting our questions out
-
-Let's think of a logical order for our questions, and how they fit together.
-
-It might make sense to first learn a little about what the person prefers. It makes
-sense to choose a question order that is respectful of the user's time. Let's think of big
-questions up front that might reduce the number of unnecessary questions we have to ask later.
-
-A logical first question might be:
-
-1. Do you like parties?
-
-If they don't like parties at all, they can probably skip this one. But just in case, a good follow-up might be:
-
-* Is there anyone there who you want to meet?
-
-Followed by
-
-* Do you have any big assigments or studying to do?
-
-If they do like parties, we might ask:
-
-2. Do you have a big assignment due?
-
-## Map it out
-
-For a small problem like this, it might be helpful to write out a small flow chart or map of the logic. For a bigger problem, you might come up with a different way to organize it.
-
-Below is a flowchart that maps out part of the logic we worked out above:
-
-[![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggVERcblx0QVtEbyB5b3UgbGlrZSBwYXJ0aWVzP10gLS0-IEJ7eWVzfVxuICBBIC0tPiBDe25vfVxuICBDIC0tPkRbSXMgdGhlcmUgYW55b25lIHlvdSB3YW50IHRvIG1lZXQ_XVxuICBEIC0tPkV7eWVzfVxuICBEIC0tPkZ7bm99XG4gIEYgLS0-IFooZmE6ZmEtYmVkIERvbid0IEdvKVxuICBFIC0tPiBHW0RvIHlvdSBoYXZlIGEgYmlnIGFzc2lnbm1lbnQgZHVlP11cbiAgRyAtLT5oe3llc31cbiAgRyAtLT5pe25vfVxuICBpIC0tPlkoZmE6ZmEtZ2lmdCBHbyB0byB0aGUgcGFydHkpXG5cbiAgaCAtLT4gWlxuICBCIC0tPiBHXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcblx0QVtEbyB5b3UgbGlrZSBwYXJ0aWVzP10gLS0-IEJ7eWVzfVxuICBBIC0tPiBDe25vfVxuICBDIC0tPkRbSXMgdGhlcmUgYW55b25lIHlvdSB3YW50IHRvIG1lZXQ_XVxuICBEIC0tPkV7eWVzfVxuICBEIC0tPkZ7bm99XG4gIEYgLS0-IFooZmE6ZmEtYmVkIERvbid0IEdvKVxuICBFIC0tPiBHW0RvIHlvdSBoYXZlIGEgYmlnIGFzc2lnbm1lbnQgZHVlP11cbiAgRyAtLT5oe3llc31cbiAgRyAtLT5pe25vfVxuICBpIC0tPlkoZmE6ZmEtZ2lmdCBHbyB0byB0aGUgcGFydHkpXG5cbiAgaCAtLT4gWlxuICBCIC0tPiBHXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
-
-## Building our Docassemble interview
 
-### What kinds of information do we need to gather?
-
-Before we start, let's think about what kinds of information we will want to ask.
-
-This Docassemble interview will use two kinds of screens:
-
-1. Yes/no questions
-2. Ending screens
-
-As a reminder, ending screens use the `event` specifier. 
```

Comment:
Note: I actually like how this lays it out. I couldn't figure out a way to make it work in my narrative structure, and maybe that's a sign of a problem with my structure. Maybe I should add it as an kind of tl;dr or something because it's a nice quick reference.